paddlepaddle/paddlepaddle-engine/src/main/java/ai/djl/paddlepaddle/jna/PD_Buffer.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+package ai.djl.paddlepaddle.jna;
+
+import com.sun.jna.Pointer;
+import com.sun.jna.Structure;
+
+/** The buffer structure defined by Paddle. */
+@SuppressWarnings({"missingjavadocmethod", "typename"})
+public class PD_Buffer extends Structure {
```

Comment:
nit: 
```suggestion
public class PDBuffer extends Structure {
```

Reply:
I prefer keep PD_Buffer

Reply:
This have to match right? Or JNA will not map this to the right structure?

Reply:
I doesn't have to match the C++ name, but just make it easy to keep them the same for reference